### DISCUSSION TITLE
override configuration

### DISCUSSION THREAD

**Discussion by DFazeli** [keycloak/keycloak#11107] [Q&A] [answered]
We have `sit env` and we want to launch keycloak on `uat env` with sit database, so we export and import keycloak database from `sit` to `uat`. when we launch `helm-chart` with new config e.g: `new broker`, `new Redis`, `new user admin and password`, not override on `uat env` and all configuration is `sit env`! how can we launch with new configuration?
I use  `https://artifacthub.io/packages/helm/codecentric/keycloak/` helm-chart

thanks a lot.

**Comment by abstractj** [comment, marked as answer]
@DFazeli we do not provide any support for the helm chart you mentioned. I'd suggest asking those questions to the project owner instead https://github.com/codecentric/helm-charts/tree/master/charts/keycloak
